.github/workflows/ci.yml

```diff
@@ -66,15 +66,15 @@ jobs:
         ./linux-container-ci.sh ${{ env.BUILDER_VERSION }} aws-crt-${LINUX_BASE_IMAGE} build -p ${{ env.PACKAGE_NAME }} --compiler=${{ matrix.compiler }}
 
   windows:
-    runs-on: windows-latest
+    runs-on: windows-2019
     steps:
     - name: Build ${{ env.PACKAGE_NAME }} + consumers
       run: |
         python -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
         python builder.pyz build -p ${{ env.PACKAGE_NAME }} --spec=downstream
 
   windows-vc14:
-    runs-on: windows-latest
+    runs-on: windows-2015
```

Comment:
there is no `windows-2015` image. also comment pls for future maintainers:

suggestion:
`runs-on: windows-2019 # windows-2019 is last env with Visual Studio 2015 (v14.0)`